commcare_connect/opportunity/views.py

```diff
@@ -497,10 +498,38 @@ def visit_verification(request, org_slug=None, pk=None):
     access_id = OpportunityAccess.objects.get(user=user_visit.user, opportunity=user_visit.opportunity).id
     serializer.is_valid()
     xform = serializer.save()
+    user_forms = []
+    other_forms = []
+    lat = None
+    lon = None
+    if user_visit.location:
+        locations = (
+            UserVisit.objects.filter(opportunity=user_visit.opportunity)
+            .values("entity_id", "location", "user_id")
+            .exclude(pk=pk)
+        )
+        lat, lon, *_ = user_visit.location.split(" ")
+        for loc in locations:
+            other_lat, other_lon, *_ = loc["location"].split(" ")
+            dist = distance.distance((lat, lon), (other_lat, other_lon))
+            if user_visit.user_id == loc["user_id"]:
+                user_forms.append((loc, dist.m, other_lat, other_lon))
+            else:
+                other_forms.append((loc, dist.m, other_lat, other_lon))
+            user_forms.sort(key=lambda x: x[1])
```

Comment:
is this sorting for forms necessary for the UI?

Reply:
yeah, we only want the 5 closest forms (you can see when we pass the list to the page that we only take the first five), so we sort them by distance here

Reply:
@calellowitz  - Note that we want the 5 closest forms within some threshold. I'm going to arbitrarily say that the threshold is 250 meters based on heuristically spot checking the data around. 

@pxwxnvermx one other thing I'm going to ask for at some point (possibly while Cal is offline) is the ability to change the actual marker images to be different (not just color) for the two types of geopoint, and to add a Legend to either the map view or next to the map view. If either of those is relevant for Cal's code here, good to plan for ahead.

Reply:
> Note that we want the 5 closest forms within some threshold. I'm going to arbitrarily say that the threshold is 250 meters based on heuristically spot checking the data around.

Updated to cap at 250 meters.

The other two will require a bit more work, but should be achievable. I can also pick up one or both on monday when I get back.